pkgs/development/libraries/yyjson/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "yyjson";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "ibireme";
+    repo = "yyjson";
+    rev = version;
+    hash = "sha256-Cz8K+cWuDpoMY6d+ecuOvXIMc4wtx15LLvxpFibkNyw=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  meta = with lib; {
+    description = "The fastest JSON library in C";
+    homepage = "https://github.com/ibireme/yyjson";
```

Comment:
This can be inherited from `src`

Reply:
this is a bad practice indeed. Nixpkgs should not rely on "macro expansion".